src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp

```diff
@@ -106,6 +106,13 @@ CorProfiler::Initialize(IUnknown* cor_profiler_info_unknown) {
     Info("Profiler is operating within Azure App Services context.");
     in_azure_app_services = true;
 
+    std::vector<WSTRING> app_insights_processes{"SnapshotUploader.exe"_W};
```

Comment:
Should we add this process name to environment variable `DD_PROFILER_EXCLUDE_PROCESSES` wherever it is we add all the other environment variables?

Reply:
I think so. We should also consider using the Raw Profiler Hook env variables of CLRIE. We're not a CLRIE profiler yet, but if we register as a Raw Profiler using the environment variables, we will be able to co-exist with CLRIE profilers like AppInsights. https://github.com/microsoft/CLRInstrumentationEngine/blob/develop/docs/getting_started.md#how-do-i-use-the-raw-profiler-hook

Reply:
Good call